libs/libvtrutil/CMakeLists.txt

```diff
@@ -83,7 +83,7 @@ add_custom_command(OUTPUT ${VTR_VERSION_FILE_OUT}
 #
 # Source files and library
 #
-file(GLOB_RECURSE LIB_SOURCES src/*.cpp)
+file(GLOB_RECURSE LIB_SOURCES src/*.cpp src/*.c)
```

Comment:
Probably easier to make the new specrand.c file into specrand.cpp --> ask Mahesh if that's OK.  If not, we can do this.

Reply:
It's up you how you want to pull in these files. If you'd like to convert to cpp, please go ahead. There is no requirement from my side :-)